### PR TITLE
Change GitHub and Homepage links

### PR DESCRIPTION
**Pull request by maciejnems**

### CODE REVIEW

File: packages/apps/src/Menu/index.tsx [hunk 1]
```diff
@@ -28,16 +28,16 @@ interface Props {
...
+      href: 'https://github.com/aleph-zero-foundation',
       icon: 'code-branch',
       name: 'github',
       text: t<string>('nav.github', 'GitHub', { ns: 'apps-routing' })
     },
     {
-      href: 'https://wiki.polkadot.network',
+      href: 'https://alephzero.org/',
       icon: 'book',
       name: 'wiki',
```
Comment (Marcin-Radecki) [inline packages/apps/src/Menu/index.tsx:39]: If we change this to `homepage`, this should break sth right? As those are likely references in some CSS files?

**Review by Marcin-Radecki — APPROVED** [review]

File: packages/apps/src/Menu/index.tsx (hunk 1 above)
Comment (maciejnems) [reply packages/apps/src/Menu/index.tsx:39]: Well, this shouldn't break anything. From what I understand we want to change it back in the future to linking to wiki. Actually I could change this and `text` to `t<string>('nav.homepage', 'Homepage', { ns: 'apps-routing' })` and everything should be working and showing "homepage" in all languages.

Comment (maciejnems) [reply packages/apps/src/Menu/index.tsx:39]: So I went with the least invasive option to change it back in the future. Is it fine with you?

Comment (Marcin-Radecki) [reply packages/apps/src/Menu/index.tsx:39]: Yes sure,

**Pull request merged**